src/app-view.js

```diff
@@ -89,17 +99,31 @@ class AppView extends React.Component {
     }
 
     handleKeyDown(event) {
-        if (event.keyCode === 83) {
+        if (event.keyCode === 83 && !this.checkSearchFieldFocus()) {
             this.setSidebarVisibility();
         }
     }
 
+    checkSearchFieldFocus() {
+        const input = this.searchFieldRef.current;
+
+        return document.activeElement === input;
+    }
+
     checkCurrentSection() {
         this.setState({
             sections: mapPropsToSections(this.props.sections, this.props.currentHash),
         });
     }
 
+    checkMobileDevice() {
```

Comment:
could you please move it in separate folder - utilities/index, where you can export this function

and cover it with some tests, please?

Reply:
Moved to utilities folder. Tests are coming.

Reply:
Tests are done